jdaviz/__init__.py

```diff
@@ -14,3 +17,9 @@
 from jdaviz.configs.cubeviz import Cubeviz, CubeViz  # noqa
 from jdaviz.configs.imviz import Imviz  # noqa
 from jdaviz.utils import enable_hot_reloading  # noqa
+
+import bqplot
+from jdaviz.core.marks import BaseUnitLine
+bqplot.Lines = BaseUnitLine
+
+del layer_artist, bqplot, BaseUnitLine
```

Comment:
Just a friendly note to NEVER let this atrocity get merged, but it's the dark magic that allows us to implement this POC in jdaviz without having to modify glue/glue-jupyter.